docker-compose.yml

```
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_USER: root
+      MYSQL_ROOT_PASSWORD: 
+    ports:
+      - "4306:3306"
+    volumes:
+      - ./db/mysql/volumes:/var/lib/mysql
+
+  web:
+    build: .
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
+    volumes:
+      - .:/KStargram
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
```

サービスの設定の変更と、永続化の設定を追加します。

```suggestion
　volumes:
      - .:/KStargram
      - bundle:/usr/local/bundle
    environment:
      RAILS_ENV: development
    ports:
      - "3000:3000"
    tty: true
    depends_on:
      - db
      
volumes:
  bundle:
    driver: local
  data:
    driver: local
```